# import function_one


# def muly(x, y):
#     print("Multiplication of x and y is: ", x * y)


# if __name__ == "__main__":
#     muly(3, 3)
#     function_one.addition(43, 43)

# ğŸŒºğŸŒºğŸŒºğŸŒºğŸŒºğŸŒºğŸŒºğŸŒºğŸŒºğŸŒºğŸŒºğŸŒºğŸŒºğŸŒºğŸŒºğŸŒºğŸŒºğŸŒºğŸŒºğŸŒºğŸŒº
# How to change current directory with exception handling challenge
# import os

# cwd = os.getcwd()
# complete_path = cwd + "\\dummy_folder"
# try:
#     os.chdir(complete_path)
#     print("Path has been changed.")
#     print(os.getcwd())

# except NotADirectoryError:
#     print(NotADirectoryError)

# except FileNotFoundError:
#     print("File not found.")
